#Позиция ключа, который хотим переключить
int8 position

# Новое значение ключа
int8 state